scrollytelling/src/primitive.tsx

```diff
@@ -109,6 +111,7 @@ const Scrollytelling = ({
         start: start ?? "top top",
         end: end ?? "bottom bottom",
         trigger: ref.current,
+        toggleActions: toggleActions ?? "play none none none",
```

Comment:
nitpick: i'd leave it like this so that it defaults to what it's specified on GSAP

```suggestion
        toggleActions,
```